From 314dd29042867f44b17a3205a1c78c5527499dcd Mon Sep 17 00:00:00 2001
From: Uri Mashiach <uri.mashiach@compulab.co.il>
Date: Tue, 3 Sep 2019 15:57:46 +0300
Subject: [PATCH 03/13] sb-imx8x: rename node imx8qxp-mek to sb-imx8x

The PINMUX node imx8qxp-mek was taken from the NXP evaluation board
device tree.

Rename the PINMUX node imx8qxp-mek to sb-imx8x.

Signed-off-by: Uri Mashiach <uri.mashiach@compulab.co.il>
---
 arch/arm64/boot/dts/compulab/sb-imx8x.dts | 84 +++++++++++++++----------------
 1 file changed, 42 insertions(+), 42 deletions(-)

diff --git a/arch/arm64/boot/dts/compulab/sb-imx8x.dts b/arch/arm64/boot/dts/compulab/sb-imx8x.dts
index bd1209056055..36f382a61478 100644
--- a/arch/arm64/boot/dts/compulab/sb-imx8x.dts
+++ b/arch/arm64/boot/dts/compulab/sb-imx8x.dts
@@ -183,50 +183,51 @@
 };
 
 &iomuxc {
-	pinctrl_usdhc2_gpio: usdhc2gpiogrp {
-		fsl,pins = <
-			SC_P_USDHC1_RESET_B_LSIO_GPIO4_IO19	0x00000021
-			SC_P_USDHC1_WP_LSIO_GPIO4_IO21		0x00000021
-			SC_P_USDHC1_CD_B_LSIO_GPIO4_IO22	0x00000021
-		>;
-	};
+	sb-imx8x {
+		pinctrl_usdhc2_gpio: usdhc2gpiogrp {
+			fsl,pins = <
+				SC_P_USDHC1_RESET_B_LSIO_GPIO4_IO19	0x00000021
+				SC_P_USDHC1_WP_LSIO_GPIO4_IO21		0x00000021
+				SC_P_USDHC1_CD_B_LSIO_GPIO4_IO22	0x00000021
+			>;
+		};
 
-	pinctrl_usdhc2: usdhc2grp {
-		fsl,pins = <
-			SC_P_USDHC1_CLK_CONN_USDHC1_CLK		0x06000041
-			SC_P_USDHC1_CMD_CONN_USDHC1_CMD		0x00000021
-			SC_P_USDHC1_DATA0_CONN_USDHC1_DATA0	0x00000021
-			SC_P_USDHC1_DATA1_CONN_USDHC1_DATA1	0x00000021
-			SC_P_USDHC1_DATA2_CONN_USDHC1_DATA2	0x00000021
-			SC_P_USDHC1_DATA3_CONN_USDHC1_DATA3	0x00000021
-			SC_P_USDHC1_VSELECT_CONN_USDHC1_VSELECT	0x00000021
-		>;
-	};
+		pinctrl_usdhc2: usdhc2grp {
+			fsl,pins = <
+				SC_P_USDHC1_CLK_CONN_USDHC1_CLK		0x06000041
+				SC_P_USDHC1_CMD_CONN_USDHC1_CMD		0x00000021
+				SC_P_USDHC1_DATA0_CONN_USDHC1_DATA0	0x00000021
+				SC_P_USDHC1_DATA1_CONN_USDHC1_DATA1	0x00000021
+				SC_P_USDHC1_DATA2_CONN_USDHC1_DATA2	0x00000021
+				SC_P_USDHC1_DATA3_CONN_USDHC1_DATA3	0x00000021
+				SC_P_USDHC1_VSELECT_CONN_USDHC1_VSELECT	0x00000021
+			>;
+		};
 
-	pinctrl_usdhc2_100mhz: usdhc2grp100mhz {
-		fsl,pins = <
-			SC_P_USDHC1_CLK_CONN_USDHC1_CLK		0x06000041
-			SC_P_USDHC1_CMD_CONN_USDHC1_CMD		0x00000021
-			SC_P_USDHC1_DATA0_CONN_USDHC1_DATA0	0x00000021
-			SC_P_USDHC1_DATA1_CONN_USDHC1_DATA1	0x00000021
-			SC_P_USDHC1_DATA2_CONN_USDHC1_DATA2	0x00000021
-			SC_P_USDHC1_DATA3_CONN_USDHC1_DATA3	0x00000021
-			SC_P_USDHC1_VSELECT_CONN_USDHC1_VSELECT	0x00000021
-		>;
-	};
+		pinctrl_usdhc2_100mhz: usdhc2grp100mhz {
+			fsl,pins = <
+				SC_P_USDHC1_CLK_CONN_USDHC1_CLK		0x06000041
+				SC_P_USDHC1_CMD_CONN_USDHC1_CMD		0x00000021
+				SC_P_USDHC1_DATA0_CONN_USDHC1_DATA0	0x00000021
+				SC_P_USDHC1_DATA1_CONN_USDHC1_DATA1	0x00000021
+				SC_P_USDHC1_DATA2_CONN_USDHC1_DATA2	0x00000021
+				SC_P_USDHC1_DATA3_CONN_USDHC1_DATA3	0x00000021
+				SC_P_USDHC1_VSELECT_CONN_USDHC1_VSELECT	0x00000021
+			>;
+		};
+
+		pinctrl_usdhc2_200mhz: usdhc2grp200mhz {
+			fsl,pins = <
+				SC_P_USDHC1_CLK_CONN_USDHC1_CLK		0x06000041
+				SC_P_USDHC1_CMD_CONN_USDHC1_CMD		0x00000021
+				SC_P_USDHC1_DATA0_CONN_USDHC1_DATA0	0x00000021
+				SC_P_USDHC1_DATA1_CONN_USDHC1_DATA1	0x00000021
+				SC_P_USDHC1_DATA2_CONN_USDHC1_DATA2	0x00000021
+				SC_P_USDHC1_DATA3_CONN_USDHC1_DATA3	0x00000021
+				SC_P_USDHC1_VSELECT_CONN_USDHC1_VSELECT	0x00000021
+			>;
+		};
 
-	pinctrl_usdhc2_200mhz: usdhc2grp200mhz {
-		fsl,pins = <
-			SC_P_USDHC1_CLK_CONN_USDHC1_CLK		0x06000041
-			SC_P_USDHC1_CMD_CONN_USDHC1_CMD		0x00000021
-			SC_P_USDHC1_DATA0_CONN_USDHC1_DATA0	0x00000021
-			SC_P_USDHC1_DATA1_CONN_USDHC1_DATA1	0x00000021
-			SC_P_USDHC1_DATA2_CONN_USDHC1_DATA2	0x00000021
-			SC_P_USDHC1_DATA3_CONN_USDHC1_DATA3	0x00000021
-			SC_P_USDHC1_VSELECT_CONN_USDHC1_VSELECT	0x00000021
-		>;
-	};
-	imx8qxp-mek {
 		pinctrl_i2c0_mipi_lvds0: mipi_lvds0_i2c0_grp {
 			fsl,pins = <
 				SC_P_MIPI_DSI0_I2C0_SCL_MIPI_DSI0_I2C0_SCL	0xc6000020
@@ -250,4 +251,3 @@
 		};
 	};
 };
-
-- 
2.11.0

